cicd/PRChecks.jenkinsfile

```
@@ -0,0 +1,51 @@
+static final RUBY_VERSION = "3.1.2"
+
+
+pipeline {
+  agent {
+    kubernetes {
+      yaml """\
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        labels:
+          some-label: dind-agent
```

Meaningful agent pod label
```suggestion
          jenkins-agent: casino-prcheck
```